### PR TITLE
chore(dates): corrected format and utc problem for invoices dates

### PR DESCRIPTION
**Pull request by rockambole86**
### Background

Since the time part is not sent anymore by the billing-api, we had to update the the way the dates are displayed, otherwise it does a UTC -> local conversion that is wrong.

### Changes done

- Set the locale option as UTC for the invoices dates
- Updated the mocked data

### CODE REVIEW

File: src/headerData.json [hunk 1]
```diff
@@ -169,5 +169,5 @@
   },
   "loginUrl": "https://app.fromdoppler.com/#/login?lang=es",
   "notifications": [],
-  "jwtToken": "eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc1N1IjpmYWxzZSwic3ViIjoiY2Jlcm5hdEBnZXRjcy5jb20iLCJjdXN0b21lcklkIjoiMTAyNCIsImRhdGFodWJDdXN0b21lcklkIjoiMTAyNCIsImlhdCI6MTU1NjU0NTQ2MSwiZXhwIjoxNTU2NTQ3MjYxfQ.meC1sFsC7m8nGFfcK2oih1hqdKQ4Lj81O5rN_-awOcM_JEe9ddviocPbYAg64L6nC6b0a7J5xmTmiW_-MAIEIg"
+  "jwtToken": "eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc1NVIjp0cnVlLCJpYXQiOjE1ODQ0NjM1MjksImV4cCI6MTYxNTU2NzUyOX0.OpfsA_5WUE5L9H8hUVqJo2_G-L2XLWy99MnxNr4rOUZWNlqywfT2f4z_WQ0BEh8ryhuOhRGcr5js6gch1gp4xj1N7jBb_QSk_nvxA2JW_VHYaGd1-ieaxDQs4lDj2qmlAgLymmwF7K55Xtdq5UhWfluRo_Gq1YY5IUnMIREApGEOAce4YsyQ_7Q0L_BSR-oDYvmMQG7HlwGsreI62_0RYY7wo_RmTLWaz8Zsh9NWfhCg8Wq6z2hl_KqvWOKplyHlwVZWrSBewM5NiKGH9a2aIRRV_sSWZYqMnN86GS1ymQWEK2pUFGyXG7eGhHHGBQg_LASMqSOSSRZP4iPjWFjCtQ"
```
Comment (andresmoschini) [inline src/headerData.json:172]: Creo que no hay que incluir este cambio, cuidado si esto es un token real.

File: src/components/InvoicesList/InvoicesList.js [hunk 2]
```diff
@@ -132,10 +136,12 @@ const InvoicesList = ({ dependencies: { dopplerBillingApiClient } }) => {
                           </td>
                           <td>{invoice.documentNumber}</td>
                           <td>
-                            <FormattedDate value={invoice.creationDate} />
+                            <FormattedDate value={invoice.creationDate} {...dateFormatOptions} />
```
Comment (andresmoschini) [inline src/components/InvoicesList/InvoicesList.js:139]: No me convence mucho manejarlo a nivel presentación, pero dado que no hay un tipo especial para fechas sin hora, no me parece mal.

File: src/components/InvoicesList/InvoicesList.js [hunk 3]
```diff
@@ -24,6 +24,10 @@ const InvoicesList = ({ dependencies: { dopplerBillingApiClient } }) => {
     maximumFractionDigits: 2,
   };
 
+  const dateFormatOptions = {
```
Comment (andresmoschini) [inline src/components/InvoicesList/InvoicesList.js:27]: What do you think about 

```suggestion
  const utdDateFormatOptions = {
```

Comment (rockambole86) [reply src/components/InvoicesList/InvoicesList.js:27]: utd or utc?

File: src/headerData.json (hunk 1 above)
Comment (rockambole86) [reply src/headerData.json:172]: Es un token con el signature de DEV, así que no habría drama; pero de todas formas lo saco.

File: src/components/InvoicesList/InvoicesList.js (hunk 3 above)
Comment (andresmoschini) [reply src/components/InvoicesList/InvoicesList.js]: Sorry, UTC.

Comment (rockambole86) [reply src/components/InvoicesList/InvoicesList.js]: updated the name to `utcDateFormatOptions`

**Pull request merged**